contexts/did-configuration-v0.0.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": [
+    "https://schema.org",
+    {
+      "@version": 1.1,
+      "didcfg": "https://identity.foundation/.well-known/contexts/did-configuration-v0.0#",
+      "DomainLinkageAssertion": "didcfg:DomainLinkageAssertion",
```

Comment:
This was raised as an issue of the CCG mailing list regarding the vc greeting card example. Apparently the type of the credential should be defined in the context... I was also confused by this.

Reply:
Could you link the supporting evidence for that? I'm not convinced because this seems in contradiction with what a context file is (mapping terms with IRIs)

Reply:
Here is the thread where this was discussed: https://lists.w3.org/Archives/Public/public-credentials/2019Dec/0006.html

You can also find similar examples here: https://www.w3.org/2018/credentials/examples/v1

For the example: https://www.w3.org/TR/vc-data-model/#example-1-a-simple-example-of-a-verifiable-credential

Although this seems to support your assertion since "AlumniCredential" is not present in the context.

Reply:
I think you are right, and I'm not seeing any error with the credential format, with "DomainLinkageAssertion" removed from the context.